chia/full_node/mempool_manager.py

```diff
@@ -469,10 +469,10 @@ async def add_spendbundle(
 
         new_item = MempoolItem(new_spend, uint64(fees), npc_result, cost, spend_name, additions, removals, program)
         self.mempool.add_to_pool(new_item)
-        now = time.time()
+        duration = time.time() - start_time
         log.log(
-            logging.DEBUG,
-            f"add_spendbundle {spend_name} took {now - start_time:0.2f} seconds. "
+            logging.DEBUG if duration < 2 else logging.WARNING,
```

Comment:
is 2 seconds for a mempool insertion a reasonable cut-off?

Reply:
FWIW, my full node on a 8 years old mac never shows this going above ~0.3 secs. What's interesting is that higher durations don't seem to be related to higher cost of bundle.

Reply:
yeah, I suspect at those times it's going to be a lot of regular computer noise. scheduling and memory cache.